nodejs/awsx/package.json

```diff
@@ -19,7 +19,7 @@
         "mime": "^2.0.0"
     },
     "peerDependencies": {
-        "@pulumi/aws": "^4.6.0",
+        "@pulumi/aws": "~4.23.0",
```

Comment:
This change works around https://github.com/pulumi/pulumi-aws/issues/1660.  It should be removed before merging if that issue is fixed.